pkg/pillar/cmd/zedagent/zedagent.go

```diff
@@ -878,6 +900,7 @@ func Run() {
 
 		case <-stillRunning.C:
 		}
+		doDeviceReboot(&zedagentCtx)
```

Comment:
I suspect the name of this function is misleading, unless you reboot the first time you go around the for loop.

Reply:
sure